vet.sh

```diff
@@ -163,4 +163,36 @@ naming.Watcher
 resolver.Backend
 resolver.GRPCLB' "${SC_OUT}"
 
+# - special golint on package comments.
+lint_package_comment_per_package() {
+  # Number of files in this go package.
+  fileCount=$(go list -f '{{len .GoFiles}}' $1)
+  if [ ${fileCount} -eq 0 ]; then
+    return 0
+  fi
+  # Number of package errors generated by golint.
+  lintPackageCommentErrorsCount=$(golint --min_confidence 0 $1 | grep -c "should have a package comment" || true)
```

Comment:
i don't think you need this `|| true` with `set +e`

Reply:
Done